### ISSUE TITLE
.gitignore: add output formats like .tiff, .ncdf etc.

### ISSUE THREAD

**Issue by przell** [Open-EO/openeo-usecases#3]
Hi all, 
should we add the common output formats we expect to receive from the processes into the .gitignore? 
I have for example *.tiff and *.ncdf included in the .gitignore in the eurac folder. 
But I have run the udf example of Wageningen and don't want to commit my local result into the repo. So currently I have to delete it manually before pushing.
What do you think?
Best, 
Peter

**Comment by m-mohr** [comment]
Do we want to also have the results here for reference at some point? Although git is not the best place for storing binary data...

**Comment by przell** [comment]
> Although git is not the best place for storing binary data...

I agree to that. Especially since the processes could produce quite big datasets. 
In theory results should be reproducible for everyone here? Otherwise we would need a different place to share reference results.
